pytuya/__init__.py

```diff
@@ -151,6 +157,7 @@ def __init__(self, dev_id, address, local_key=None, dev_type=None, connection_ti
         self.local_key = local_key.encode('latin1')
         self.dev_type = dev_type
         self.connection_timeout = connection_timeout
+        self.version = 3.1
```

Comment:
Worth considering adding another constant for numbers to match PROTOCOL_VERSION_BYTES_31, rather that using literals